src/webui/src/static/const.ts

```diff
@@ -35,8 +35,10 @@ const COLUMN = ['Trial No.', 'ID', 'Duration', 'Status', 'Default', OPERATION];
 // all choice column !dictory final
 const COLUMNPro = ['Trial No.', 'ID', 'Start Time', 'End Time', 'Duration', 'Status',
 'Intermediate result', 'Default', OPERATION];
+const  CONCURRENCYTOOLTIP = 'TrialConcurrency is the number of trials running concurrently, which is the number of GPUs you want to use if you are setting gpuNum to 1.';
```

Comment:
Suggest: Trial concurrency is the number of trials running concurrently, which is the number of GPUs you want to use if you set gpuNum to 1.

Reply:
OK!

Reply:
`which` is a little misleading here.

I think we can just keep the former half: Trial concurrency is the number of trials running concurrently.

Reply:
I think if only show `Trial concurrency is the number of trials running concurrently.` user still don't understand `concurrency` meaning. Depend on you!

Reply:
Finally， this tooltip changed to `Trial concurrency is the number of trials running concurrently.`  by Quanlu's comments.